hydro_test/src/cluster/paxos.rs

```diff
@@ -51,16 +52,17 @@ impl PartialOrd for Ballot {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug)]
-struct LogValue<P> {
-    ballot: Ballot,
-    value: Option<P>, // might be a hole
+pub struct LogValue<P> {
+    pub ballot: Ballot,
+    pub value: Option<P>, // might be a hole
 }
 
 #[derive(Serialize, Deserialize, PartialEq, Eq, Clone, Debug)]
-struct P2a<P> {
-    ballot: Ballot,
-    slot: usize,
-    value: Option<P>, // might be a re-committed hole
+pub struct P2a<P, S> {
```

Comment:
Is this used anymore?

Reply:
Yup still used by proxy leaders to tell acceptors who to reply to